docs(api): additional 2.15 features on Versioning page (#13718)
OpenTrons/opentrons-api: api/docs/v2/versioning.rst

  
  - The API does not enforce placement restrictions for the Heater-Shaker module on Flex, because it is installed below-deck in a module caddy. Pipetting restrictions are still in place when the Heater-Shaker is shaking or its labware latch is open.
  
  - The new :py:meth:`.configure_for_volume` method can place Flex 50 µL pipettes in a low-volume mode for dispensing very small volumes of liquid. See :ref:`pipette-volume-modes`. 
  

>   - The new ``push_out`` parameter of :py:meth:`.dispense` can also help ensure that the pipette dispenses all of its liquid when working with very small volumes.

Reviewer: A suggestion:

```suggestion
  - The new ``push_out`` parameter of the :py:meth:`.dispense` method helps ensure the pipette dispenses all of its liquid when working with tiny amounts of liquid.
```
